### DISCUSSION TITLE
New style/theme with album colors for texts too?

### DISCUSSION THREAD

**Discussion by Zephyr0ck** [TT-ReBORN/Georgia-ReBORN#98] [Ideas]
On jailbroken devices there is a tweak/addon to be able to color the text based on the album art colors.

Tweak is:

**ColorFlow 4**

Could Georgia-ReBORN do the same for the text in the Details panel?

Samples:

![photo_2022-11-30_15-30-53](https://user-images.githubusercontent.com/66889502/204912370-6b8a0130-2a4f-470c-be9d-ddc215d25d09.jpg)
![photo_2022-11-30_15-30-57](https://user-images.githubusercontent.com/66889502/204912376-5025ee4b-fc46-4b48-8dee-0163235ad2bb.jpg)
![photo_2022-11-30_15-31-01](https://user-images.githubusercontent.com/66889502/204912378-d8b1caa6-60bd-42ed-a69b-72f327c1685f.jpg)
![photo_2022-11-30_15-31-04](https://user-images.githubusercontent.com/66889502/204912381-cbb656ba-bfb6-43bd-8caa-b0d6650714b4.jpg)
![photo_2022-11-30_15-31-08](https://user-images.githubusercontent.com/66889502/204912382-34830f05-6e06-4780-83bc-7eaffcb50566.jpg)
![photo_2022-11-30_15-30-49](https://user-images.githubusercontent.com/66889502/204912385-3b6aead1-84e0-464f-a286-3405a05c0b5f.jpg)

**Comment by TT-ReBORN** [comment]
Hi José,

as you know the Reborn theme already applies album art color for the whole theme.
So for the text, I could think about making accents from shading ( darken ) or tinting ( lighten ) the chosen album art color.
I've first tried to play around a bit with using the complementary color from the chosen album art color, but I'm not yet satisfied.
I will think about this, will see how it goes...

-Tom
